data/org.gnome.Hamster.metainfo.xml.in

```diff
@@ -44,6 +44,10 @@
   </screenshots>
   ​<translation type="gettext">hamster</translation>
   <url type="homepage">https://github.com/projecthamster/hamster/wiki</url>
+  <releases>
+    <!-- Provide a (clearly) fake date, otherwise flatpak will ignore the entry -->
+    <release version="@VERSION@" date="1980-01-01" />
```

Comment:
date should be an easter egg: "2007-08-01" Toms first commit to the repo

Reply:
I'm afraid that such a date might actually look like its meaningful, 1980-01-01 is even more clearly invalid/fake. I'm not entirely sure where this date will actually show up, though, but I'm inclined to leave it as is?